### DISCUSSION TITLE
Weekend PC troubles

### DISCUSSION THREAD

**Discussion by gonetz** [gonetz/GLideN64#2646] [Show and tell]
Last Saturday my desktop PC suddenly stopped to boot:
"Something is wrong with your system. Your PC needs rebooting. We will do it for you." - reboot - the same.
Ok, I have quite recent recovery image, lets recover! Bad idea. Recovery failed and my sdd became unreadable.
I lost everything on my system disk, including my N64 folder with emus, saves etc.
I created boot USB drive with installer and installed Windows from scratch. It booted successfully, so the initial problem was not caused by a hardware failure. I wanted my old good system disk back. I made another attempt: load from USB, open system repair tools, recover system from image - "we can't find recovery image for your system". Why? It is here, on another drive, in WindowsImageBackup folder! No, Windows can't see it. Luckily, I found an instruction, how to do recovery using command line tools. It was  a long process, the recovery image was quite large. But the method works! After many many hours I successfully applied the image to my sdd. Reboot, loading system - "Something is wrong with your system. Your PC needs rebooting. We will do it for you." WTF?!! I just successfully applied disk image, which was taken when the system disk was in a valid state! I attached the drive to another PC. Yes, it is readable, all my files are back. I made a backup just in case. I attached the drive back to my desktop and somehow opened system repair menu. Recovery from an image was obviously not an option. I decided to use "revert recent system modifications". The reversion is started and, of course, failed. PC rebooted, system started to boot, said that attempt to revert the modifications was unsuccessful (yep, it was!) and it will roll the changes back. Auto-reboot, the same message. Another auto reboot and the system loaded! I could not believe it!!! I entered my account - everything is on its place. Sunday, 11.50 PM. It was a very hard weekend.

**Comment by oddMLan** [comment]
That sounds like either Windows applied a buggy update that bricked your system (wouldn't be the first time), OR the SSD is on its last legs. I'm suspecting is the latter because it's very weird the recovery image failed. What does [CrystalDiskInfo](https://crystalmark.info/en/software/crystaldiskinfo/) tell you about the SSD?

**Reply by gonetz** [reply]
> What does [CrystalDiskInfo](https://crystalmark.info/en/software/crystaldiskinfo/) tell you about the SSD?

Good. 99%

**Comment by Jj0YzL5nvJ** [comment]
Sounds like an MBR/GPT malformation to me. Windows can boot from a malformed partition table, but the recovery console is unable to detect where it booted itself from. If a minor problem invokes the recovery console, you end up stuck in an infinite loop =P

SSD is incredibly unreliable when modifying/verifying data without a file system (RAW mode).
